### PR TITLE
Temlate Plicy and My Orders COnsole error fix

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Template Policy and My Orders console error fix

### CODE REVIEW

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/product-page-template/policies/.content.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
                 jcr:primaryType="nt:unstructured"
                 sling:resourceType="wcm/core/components/policies/mapping"/>
             <responsivegrid
-                cq:policy="wcm/foundation/components/responsivegrid/policy_600116016309972"
+                cq:policy="wcm/foundation/components/responsivegrid/policy_6302423511085"
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/product-page-template/policies/.content.xml:17]: @DeepaliAgrwl  wht is the change here?

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
                     sling:resourceType="wcm/core/components/policies/mapping"/>
             </header>
             <responsivegrid
-                cq:policy="wcm/foundation/components/responsivegrid/policy_600116016309972"
+                cq:policy="wcm/foundation/components/responsivegrid/policy_6302423511085"
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml:27]: @DeepaliAgrwl  What is the change here?

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/product-page-template/policies/.content.xml (hunk 1 above)
Comment (DeepaliAgrwl) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/product-page-template/policies/.content.xml:17]: policy_6302423511085 is the actual hclecomm content policy that QA needs to select . Added the mapping to this policy.

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml (hunk 2 above)
Comment (DeepaliAgrwl) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml:27]: policy_6302423511085 is the actual hclecomm content policy that QA needs to select . Added the mapping to this policy.

**Pull request merged**